model/peer/registry.yaml

```diff
@@ -13,3 +13,18 @@ groups:
           of the remote service. SHOULD be equal to the actual `service.name`
           resource attribute of the remote service if any.
         examples: "AuthTokenCache"
+        note: >
+          It MUST be possible to configure the peer service name in declarative configuration as follows:
```

Comment:
please add  `Development` status for this note

Reply:
the whole attribute has `Development` stability